<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mdtSqlConnectionNameWidget</class>
 <widget class="QWidget" name="mdtSqlConnectionNameWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>50</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="lbConnectionNames">
     <property name="text">
      <string>Open connections:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="cbConnectionNames"/>
   </item>
   <item>
    <widget class="QToolButton" name="tbRemoveConnection">
     <property name="toolTip">
      <string>Remove connection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="list-remove">
       <normaloff>../../../../</normaloff>../../../../</iconset>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="tbAddConnection">
     <property name="toolTip">
      <string>Add a connection</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset theme="list-add">
       <normaloff>../../../../</normaloff>../../../../</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>51</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
